docs/_data/docstoc.yml

```diff
@@ -31,8 +31,8 @@
           url: mdt-vsc-overview.html
         - title: 'Installing Codewind for VS Code'
           url: mdt-vsc-installinfo.html
-        - title: 'Using a template repository of your own'
-          url: mdt-vsc-usingadifferenttemplate.html
+        - title: 'Working with templates'
```

Comment:
This part belongs to Codewind for VS Code, you can let @tetchel review.

Reply:
Tim has already reviewed on PR https://github.com/eclipse/codewind-docs/pull/208 - for some reason the changes didn't appear in my local copy of the docs so I added them in again, just to be sure.